Uploading new covers shouldn't have an original URL character limit

**Describe the bug**
```
Caused by: org.springframework.dao.DataIntegrityViolationException: could not execute statement [Value too long for column "ORIGINAL_URL CHARACTER VARYING(255)": "'https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEj0JwtfuSpiBkLf8bZJZK... (286)"; SQL statement:

insert into image (content_id,content_length,mime_type,original_url,type,id) values (?,?,?,?,?,?) [22001-232]] [insert into image (content_id,content_length,mime_type,original_url,type,id) values (?,?,?,?,?,?)]; SQL [insert into image (content_id,content_length,mime_type,original_url,type,id) values (?,?,?,?,?,?)]
```
when attempting to use a custom image, the options are to search steamgriddb or provide a URL. The problem is that it seems to save the original URL to the database with a column size that's too few characters for some urls, making it impossible to update without saving the image and reuploading elsewhere since you can't upload directly.

Uploading from your computer would be a fine additional option, but in general I think expanding the size of that column or allowing for some more graceful error handling would be a good way to solve this issue in the interim.

--- a/app/src/main/kotlin/org/gameyfin/app/media/Image.kt
+++ b/app/src/main/kotlin/org/gameyfin/app/media/Image.kt
@@ -1,9 +1,6 @@
 package org.gameyfin.app.media
 
-import jakarta.persistence.Entity
-import jakarta.persistence.GeneratedValue
-import jakarta.persistence.GenerationType
-import jakarta.persistence.Id
+import jakarta.persistence.*
 import org.springframework.content.commons.annotations.ContentId
 import org.springframework.content.commons.annotations.ContentLength
 import org.springframework.content.commons.annotations.MimeType
@@ -14,6 +11,7 @@ class Image(
     @GeneratedValue(strategy = GenerationType.AUTO)
     var id: Long? = null,
 
+    @Lob
     val originalUrl: String? = null,
 
     val type: ImageType,

--- a/app/src/main/resources/db/migration/V2.3.0.7__Disable_length_limit_for_image_original_url.sql
+++ b/app/src/main/resources/db/migration/V2.3.0.7__Disable_length_limit_for_image_original_url.sql
@@ -1,0 +1,8 @@
+-- Flyway Migration: V2.2.0.7
+-- Purpose: Make IMAGE.ORIGINAL_URL column unbounded to avoid length errors when storing large encrypted values.
+-- Context: Previously defined as CHARACTER VARYING(255); H2 raised 22001 (value too long).
+-- Strategy: Alter column type to CLOB (unlimited length in H2). This matches other large text usages (e.g., COMMENT, SUMMARY) which use CHARACTER LARGE OBJECT.
+
+ALTER TABLE IMAGE
+    ALTER COLUMN ORIGINAL_URL CLOB;
+

